docs/integrator/options.md

```diff
@@ -56,6 +56,13 @@ No Default Value
 
 Use this to override detection of if the content video is a live stream. Live detection checks if the duration is `Infinity` but there are cases when this check is insufficient.
 
+### allowVjsAutoplay
+
+Type: `boolean`
+Default Value: `false`
+
+Set this to `true` if you intend to use video.js's custom autoplay settings ("play", "muted", or "any").
```

Comment:
Thought: should we assume `allowVjsAutoplay` if `normalizeAutoplay` is set?

Reply:
Specifically, since they're basically intended to work together.

Reply:
Yeah, that makes sense to me

Reply:
Updated